src/main/java/com/authservice/authservice/service/UserDetailsServiceImpl.java

```diff
@@ -30,10 +31,17 @@ public UserDetailsServiceImpl(UserDetailsDao userDetailsDao) {
     @Override
     public UserDetails loadUserByUsername(String userName) throws UsernameNotFoundException {
         log.info("Entering UserDetailsServiceImpl.loadUserByUsername with parameter userName {}.", userName);
-        User user = userDetailsDao.getUserByUsername(userName);
-        if (user == null) {
+        User[] users = userDetailsDao.getUserByUsername(userName);
+        if (users == null && users.length == 0)
```

Comment:
why would there be a `&&` it will never be null AND users.length == 0

Reply:
Done.